recipes/rsw-ha/templates/justfile

```diff
@@ -114,6 +114,13 @@ add-user name password:
 mount-efs:
     sudo mkdir -p /mnt/efs;
     sudo mount -t efs -o tls {{EFS_ID}}:/ /mnt/efs;
+    #!/bin/bash
+    sudo bash -c 'cat <<EOF > /etc/fstab
+    # mount efs
+    {{EFS_ID}}:/ /etc/fstab efs defaults,_netdev 0 0
+    
+
+    EOF'
```

Comment:
I am getting an error on this command. With justfile I don't think you can have a *shebang* in the middle of a "script". This part:

```
    #!/bin/bash
    sudo bash -c 'cat <<EOF > /etc/fstab
    # mount efs
    {{EFS_ID}}:/ /etc/fstab efs defaults,_netdev 0 0


    EOF'
```

Should be its own just command.

Reply:
Fixed in caec0fd